.github/workflows/ci-tests.yml

```diff
@@ -3,7 +3,7 @@ name: 💚 CI Tests
 on:
   push:
     branches:
-      - "**"
+      - '**'
```

Comment:
If you're touching this line, might as well get rid of the filter entirely, i.e. just keep:

```
on:
  push:
```